drivers/Kconfig.net

```diff
@@ -6,5 +6,6 @@
 
 menu "Network Device Drivers"
 rsource "mrf24j40/Kconfig"
+rsource "hdc1000/Kconfig"
```

Comment:
Please, add a new menu `Sensor Device Drivers`, and place this inside.

Reply:
Sure.